system_baseline/internal_views/v1.py

```diff
@@ -46,3 +47,35 @@ def get_baselines_by_system_id(system_id=None):
     current_app.logger.audit(message, request=request, success=True)
 
     return [result.system_baseline_id for result in query_results]
+
+
+@metrics.baseline_delete_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def delete_systems_by_ids(system_ids):
+    """
+    delete a list of systems given their system IDs as a list
+    """
+    ensure_rbac_write()
```

Comment:
We shouldn't need to add ensure rbac write on the internal endpoint.  We have no user when we're internal, so there shouldn't be any rbac permissions involved, this is our shared secret work instead.

Reply:
You are right, we don't need it right now. But ensure_rbac_read() is called anyway before each request lands here and both of these methods calls code to check shared secret present.

I would be more happier to keep it there as it kind of says what is the method doing (modifying something) and it is future proof when we will move this to public APIs.

Reply:
Removed